bootstrap/src/main/scala/org/scalafmt/bootstrap/Bootstrap.scala

```diff
@@ -77,7 +77,7 @@ object ScalafmtBootstrap {
         logger = Some(logger)
```

Comment:
Can you please remove the `cache` file, since it didn't fix the issue. I prefer to reuse the default coursier cache.